ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/ConnectIntegrationTest.java

```diff
@@ -223,6 +226,40 @@ public void shouldDropConnector() {
         is("mock-connector"));
   }
 
+  @Test
+  public void shouldCreateSourceConnectorSuccessfully() {
```

Comment:
nit: rename test to be consistent with the others in the file (and similarly for the other new tests too)
```suggestion
  public void shouldCreateSourceConnector() {
```